cypher-codemirror/README.md

```diff
@@ -3,11 +3,34 @@ Cypher Codemirror
 
 Package provides Cypher Editor Support integration with [CodeMirror](https://codemirror.net/) editor.
 
-### Development
+### Getting started
+Ensure that yarn is installed globally: 
 
-1. Ensure that yarn is installed globally: `npm install -g yarn`
-2. Install project dependencies: `yarn install`
+```sh
+# npm install -g yarn
```

Comment:
Do not use '#' sign in beginning. It is rendered as comment, because '#' represenst comments in shell scripts.

If you want to denote start of CLI command use '$'